### PR TITLE
Add Apache Iceberg blog post

### PR DESCRIPTION
**Pull request by ryandoowan**

### CODE REVIEW

**Comment by ryandoowan** [comment]
@mosabua @colebow @bitsondatadev

File: _posts/2023-06-30-trino-fest-2023-apacheiceberg.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "CDC patterns in Apache Iceberg"
+author: "Ryan Blue, Ryan Duan"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/ApacheIceberg.png
+show_pagenav: false
+---
+
+Have you ever needed to keep your data all in a table and an efficient way to
```
Comment (mosabua) [inline _posts/2023-06-30-trino-fest-2023-apacheiceberg.md:10]: that sentence reads weirdly

File: _posts/2023-06-30-trino-fest-2023-apacheiceberg.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+interact with them? Iceberg, an open standard for tables with SQL behavior, is
```
Comment (mosabua) [inline _posts/2023-06-30-trino-fest-2023-apacheiceberg.md:11]:
```suggestion
interact with them? Iceberg, an open standard table format, is
```

File: _posts/2023-06-30-trino-fest-2023-apacheiceberg.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+exactly what you need. But what makes Iceberg unique is its use of change data
```
Comment (mosabua) [inline _posts/2023-06-30-trino-fest-2023-apacheiceberg.md:12]:
```suggestion
exactly what you need. What makes Iceberg unique is its use of change data
```

File: _posts/2023-06-30-trino-fest-2023-apacheiceberg.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+* Read requirements with the different types of deletes in a table
+
+With these types of problems, the importance of the trade-offs between the
+different patterns rises due to the need for utmost efficiency. The first
+trade-off that Ryan talks about is the storage trade-offs between using direct
+writes and a change log table, which is considered the most important and often
+overlooked decision. The next trade-off is in regards to the MERGE pattern's
+choice of lazy merge (merge-on-read) or eager merge (copy-on-write). In
+addition, the commit frequency trade-off has different benefits depending if you
+prefer it to be faster or slower. The Flink UPSERT pattern is an update type
```
Comment (mosabua) [inline _posts/2023-06-30-trino-fest-2023-apacheiceberg.md:52]: Remove the Flink stuff

File: _posts/2023-06-30-trino-fest-2023-apacheiceberg.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+trade-off, but is not recommended due to its many drawbacks. The change log
+pattern and MERGE pattern both have benefits you may want, so Ryan suggests
+using a hybrid version of both that may give you what you want from both
+patterns. All of these different options create numerous variations depending on
+what you prefer.
+
```
Comment (mosabua) [inline _posts/2023-06-30-trino-fest-2023-apacheiceberg.md:58]: Do we need some sort of "watch the video and slides for more details" @colebow  .. I kinda of think maybe yes.. but also maybe not .. 

How about just a final closure sentence?

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _posts/2023-06-30-trino-fest-2023-apacheiceberg.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+title: "CDC patterns in Apache Iceberg"
+author: "Ryan Blue, Ryan Duan"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/ApacheIceberg.png
+show_pagenav: false
+---
+
+Have you ever wanted to keep your data in a table and have an efficient way to
+interact with them? Iceberg, an open standard table format, is
+exactly what you need. What makes Iceberg unique is its use of change data
```
Comment (mosabua) [inline _posts/2023-06-30-trino-fest-2023-apacheiceberg.md:12]:
```suggestion
exactly what you need. On of the great and unique features of the Iceberg tableformat is its support for of change data
```

File: _posts/2023-06-30-trino-fest-2023-apacheiceberg.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+capture (CDC). Co-creator of Apache Iceberg, Ryan Blue, gave a talk at [Trino Fest 2023]({% post_url
```
Comment (mosabua) [inline _posts/2023-06-30-trino-fest-2023-apacheiceberg.md:13]:
```suggestion
capture (CDC). Co-creator of Apache Iceberg, Ryan Blue, presented at [Trino Fest 2023]({% post_url
```

File: _posts/2023-06-30-trino-fest-2023-apacheiceberg.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+2023-06-20-trino-fest-2023-recap %}) this past week detailing the trade-offs
```
Comment (mosabua) [inline _posts/2023-06-30-trino-fest-2023-apacheiceberg.md:14]:
```suggestion
2023-06-20-trino-fest-2023-recap %}) this past week detailing the CDC support and the trade-offs
```

File: _posts/2023-06-30-trino-fest-2023-apacheiceberg.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+between each pattern that can be used for writing CDC streams into Iceberg
```
Comment (mosabua) [inline _posts/2023-06-30-trino-fest-2023-apacheiceberg.md:15]:
```suggestion
between different patterns that can be used for writing CDC streams into Iceberg
```

File: _posts/2023-06-30-trino-fest-2023-apacheiceberg.md [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+
+{% youtube GM7EvRc7_is %}
+
+<a class="btn btn-pink btn-md" target="_blank" href="/assets/blog/trino-fest-2023/TrinoFest2023Iceberg.pdf">
+  Check out the slides!
+</a>
+
+## Recap
+
+To begin, what is CDC and why should you use it? CDC is the idea that relational
```
Comment (mosabua) [inline _posts/2023-06-30-trino-fest-2023-apacheiceberg.md:28]:
```suggestion
To begin, what is CDC and why should you use it? CDC is the idea that when relational
```

File: _posts/2023-06-30-trino-fest-2023-apacheiceberg.md [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+or transactional tables are modified, you emit an update stream to keep copies
```
Comment (mosabua) [inline _posts/2023-06-30-trino-fest-2023-apacheiceberg.md:29]:
```suggestion
or transactional tables are modified, you emit an update stream. This enables you to keep copies
```

File: _posts/2023-06-30-trino-fest-2023-apacheiceberg.md [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+in sync so that you capture changes to tables as they happen. As Ryan states,
```
Comment (mosabua) [inline _posts/2023-06-30-trino-fest-2023-apacheiceberg.md:30]:
```suggestion
in sync by capturing changes to tables as they happen. As Ryan states,
```

File: _posts/2023-06-30-trino-fest-2023-apacheiceberg.md (hunk 5 above)
Comment (mosabua) [reply _posts/2023-06-30-trino-fest-2023-apacheiceberg.md:56]: Maybe something like

He mentions that with Iceberg you have the choice and the different CDC patterns can be supported and you can adjust your usage to your specific needs. Watch the video and review the slides for more details.

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _posts/2023-06-30-trino-fest-2023-apacheiceberg.md [hunk 13]
```diff
@@ -0,0 +1,61 @@
...
+
+* Lower latency means more work
+* Write amplification - the work necessary to balance the trade-off between
+  efficiency at write time and efficiency at read time
+* Batch writes with double update and possible inconsistency
+* Read requirements with the different types of deletes in a table
+
+With these types of problems, the importance of the trade-offs between the
+different patterns rises due to the need for utmost efficiency. The first
+trade-off that Ryan talks about is the storage trade-offs between using direct
```
Comment (colebow) [inline _posts/2023-06-30-trino-fest-2023-apacheiceberg.md:47]: Need to decide whether this is singular or plural. "The first trade-off... is the trade-offs" doesn't work.

File: _posts/2023-06-30-trino-fest-2023-apacheiceberg.md [hunk 14]
```diff
@@ -0,0 +1,61 @@
...
+writes and a change log table, which is considered the most important and often
+overlooked decision. The next trade-off is in regards to the MERGE pattern's
+choice of lazy merge (merge-on-read) or eager merge (copy-on-write). In
+addition, the commit frequency trade-off has different benefits depending if you
+prefer it to be faster or slower. The change log pattern and MERGE pattern both
```
Comment (colebow) [inline _posts/2023-06-30-trino-fest-2023-apacheiceberg.md:52]:
```suggestion
prefer it to be faster or slower. The change log pattern and `MERGE` pattern both
```
Ticks will give it syntax highlighting.

**Review by mosabua — APPROVED** [review]

**Pull request merged**